build.sbt

```diff
@@ -67,7 +70,7 @@ lazy val scalaApiModels = project.in(file("models") / "scala")
         runClean,
         runTest,
         setReleaseVersion,
-        releaseStepCommand("publishSigned"),
+        releaseStepCommandAndRemaining("+publishSigned"),
```

Comment:
Changed the release command to use `releaseStepCommandAndRemaining`, as this is required for cross-compilation: https://github.com/xerial/sbt-sonatype#using-with-sbt-release-plugin